test/KaleSpec.hs

```diff
@@ -56,12 +56,12 @@ spec = do
 
     describe "mkCommandSum" $ do
         it "is empty on empty task lists" $
-            mkCommandSum [] `shouldBe` ""
+            unCommandSumType (mkCommandSum []) `shouldBe` ""
         it "lists tasktnames properly" $
-            mkCommandSum [task0, task1]
+            unCommandSumType (mkCommandSum [task0, task1])
                 `shouldBe` concat
                     [ "data Command = Name { foo :: Int }"
-                    , " | Other_Name deriving (Eq, "
+                    , " | OtherName deriving (Eq, "
```

Comment:
I want the underscores in the name. That way the commands translate to `other-name` instead of `othername`.

Reply:
Ah - I see. Fixing now